docs/supplemental-ui/partials/header-content.hbs

```diff
@@ -28,9 +28,9 @@
             <a class="navbar-item" href="https://github.com/cerbos/cerbos">Code</a>
             <a class="navbar-item" href="https://github.com/cerbos/cerbos/issues">Issue Tracker</a>
             <a class="navbar-item" href="https://github.com/cerbos/cerbos/discussions">Discussion Forum</a>
-            <div class="navbar-item"><strong>Cerbos Cloud</strong></div>
-            <a class="navbar-item" href="https://cerbos.cloud">Sign up</a>
-            <a class="navbar-item" href="https://docs.cerbos.dev/cerbos-cloud/latest/index.html">Documentation</a>
+            <div class="navbar-item"><strong>Cerbos Hub</strong></div>
+            <a class="navbar-item" href="https://hub.cerbos.cloud">Sign up</a>
+            <a class="navbar-item" href="https://docs.cerbos.dev/cerbos-hub/latest/index.html">Documentation</a>
```

Comment:
The URL for this (currently) is just https://docs.cerbos.dev/cerbos-hub/

`/latest/` doesn't exist